src/AbstractController.php

```diff
@@ -11,65 +9,81 @@
 
 abstract class AbstractController
 {
-    /** @const string API_URL */
+    /** @const API_BASE_URI */
     private const API_BASE_URI = 'https://api.twitter.com/2/';
 
     /**
-     * @var string
+     * @const API_METHODS
+     * TODO: The HTTP method could be defined from within child controllers,
+     *       in order not having to pass the method with ->performRequest().
      */
-    private $access_token;
+    protected const API_METHODS = ['GET', 'POST', 'PUT', 'DELETE'];
```

Comment:
When the child controllers would set up the HTTP request method (which varies per API method), one doesn't have to remember and pass them. Before it was just an occasional `POST`, but these `users/{id}` endpoints use `GET`, `POST`, `PUT` and `DELETE`.